ods_ci/tests/Resources/Common.robot

```diff
@@ -305,3 +305,38 @@ Extract URLs From Text
     [Arguments]    ${text}
     ${urls}=     Get Regexp Matches   ${text}   (?:(?:(?:ftp|http)[s]*:\/\/|www\.)[^\.]+\.[^ \n]+)
     RETURN    ${urls}
+
+Run And Watch Command
+  [Documentation]    Run any shell command (including args) with optional:
+  ...    Timeout in minutes: 10 by default.
+  ...    Expected text: Verify the text exists in command output.
+  [Arguments]    ${command}    ${timeout_min}=10    ${expected_text}=${NONE}
+  Log    Watching command output: ${command}   console=True
+  @{args} =    Split String    ${command}
+  ${process_log} =    Set Variable    ${OUTPUT DIR}/${args}[0].log
```

Comment:
I would recommend against using the log name "cd.log" as it is not considered a good practice. Instead, I suggest using a more descriptive and relevant name, such as "uninstall.log" or something similar. Using meaningful and contextual log names helps improve readability and clarity when reviewing or analyzing log file.

Also, If you agree with above suggestion i don't think there is a need of Split string in line no 315

Reply:
@manosnoam let me know you though on it

Reply:
Makes sense! I changed it to set the log name according to the test name.